tlatools/org.lamport.tlatools/src/tlc2/tool/liveness/LiveCheck.java

```diff
@@ -575,7 +587,11 @@ private class TableauLiveChecker extends AbstractLiveChecker {
 		public TableauLiveChecker(OrderOfSolution oos, int soln, IBucketStatistics statistics, ILivenessStateWriter writer)
 				throws IOException {
 			super(oos, writer);
-			this.dgraph = new TableauDiskGraph(metadir, soln, statistics);
+			if (Boolean.getBoolean(LiveCheck.class.getName() + ".debug")) {
```

Comment:
it would be nice to document this anywhere...

Reply:
I'm not sure if this commit should be part of this PR because it removes the final modifier from three methods.

Reply:
I added a code comment because this is a development-time feature.